bfdd/control.c

```diff
@@ -435,6 +435,15 @@ static void control_read(struct thread *t)
 		return;
 	}
 
+#define FRR_BFD_MAXLEN  10 * 1024
+
+	if (plen > FRR_BFD_MAXLEN) {
+		zlog_debug("%s: client closed, invalid message length: %d",
+			   __func__, bcm.bcm_length);
```

Comment:
I don't know the current status of what we decided about `__func__`, but the idea was to drop `__func__` from the code everywhere which is not needed (after https://github.com/FRRouting/frr/pull/10656).

Reply:
that PR about editing every debug (more or less) in the codebase is still open?
until we resolve that question, we should continue with our existing convention (imo)